cpp/common/src/codingstandards/cpp/rules/identifierhidden/IdentifierHidden.qll

```diff
@@ -14,7 +14,10 @@ Query getQuery() { result instanceof IdentifierHiddenSharedQuery }
 query predicate problems(UserVariable v2, string message, UserVariable v1, string varName) {
```

Comment:
Let's test this rule carefully for performance, because it is known to be problematic.

Reply:
yeah noted, it has ~30 sec on openpilot for me so not the fastest